pkg/acs/calacs/Updates

```diff
@@ -49,10 +49,10 @@ Update for version 8.2.0 - 12-Aug-13 (PLL)
     acscte/getctesw.c
     calacs/acsinit.c
     calacs/calacs.c
-    calacs/calacs.h
+    calacs/acsinfo.h
```

Comment:
Why are you changing old history? You should have created your own entry at the top.